From a65adcd42d3a8d349b05f36a881b27f34bceb1fa Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Fri, 4 Nov 2011 10:31:46 -0500
Subject: [PATCH 0569/2463] ENGR00161484 Add HDMI audio support on MX6
 Sabre-lite

Add support for HDMI audio on MX6 Sabre-lite

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabrelite.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabrelite.c b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
index c39d94b..e168ccd 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabrelite.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
@@ -861,6 +861,9 @@ static void __init mx6_sabrelite_board_init(void)
 	imx6q_add_dma();
 
 	imx6q_add_dvfs_core(&sabrelite_dvfscore_data);
+
+	imx6q_add_hdmi_soc();
+	imx6q_add_hdmi_soc_dai();
 }
 
 extern void __iomem *twd_base;
-- 
1.7.7.4

